scapy/contrib/roce.py

```diff
@@ -12,14 +12,19 @@
 
 from scapy.packet import Packet, bind_layers, Raw
 from scapy.fields import ByteEnumField, ByteField, XByteField, \
-    ShortField, XShortField, XLongField, BitField, XBitField, FCSField
+    ShortField, XIntField, XShortField, XLongField, BitField, \
+    XBitField, FCSField
 from scapy.layers.inet import IP, UDP
 from scapy.layers.l2 import Ether
 from scapy.compat import raw
 from scapy.error import warning
 from zlib import crc32
 import struct
-from scapy.compat import Tuple
+from scapy.compat import (
+    Optional,
+    Tuple,
+    Type,
+)
```

Comment:
Thanks for typing this ! If you have the time, feel free to type the rest of this module and add it to `.config/mypy/mypy_enabled.txt` for it to be checked with `tox -e mypy`

Reply:
I think it is already there since 85abf7d32a65dccbaa38f9f23415e91e861d83c1.